bpaotu/bpaotu/frontend/src/components/samples_map.tsx

```diff
@@ -32,6 +32,34 @@ const ArcGIS = {
     '&amp;copy; i-cubed, USDA, USGS, AEX, GeoEye, Getmapping, Aerogrid, IGN, IGP, UPR-EGP, and the GIS User Community'
 }
 
+// tslint:disable-next-line:max-classes-per-file 
+class BPAImages extends React.Component<any, any> {
```

Comment:
rather than turning off the linter, should you make new files for these componets?

Reply:
I asked @sztamas about it and he suggested doing this. :grin:

Reply:
But happy to work with any suggestion.

Reply:
Following @sztamas suggestion.